app/generate_csv_eventyay.py

```diff
@@ -1,7 +1,12 @@
 #!usr/bin/python3
-from urllib2 import urlopen
 import json
 import os
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen
+except ImportError:
+    # Fall back to Python 2's urllib2
+    from urllib2 import urlopen
```

Comment:
Best would be to put this in a separate pull-requets. Thus, if there are any discussions here about anything else,

- this gets merged asap
- other people can build on it

Reply:
@niccokunzmann ok